src/content/github/file/main.tsx

```diff
@@ -232,40 +269,6 @@ async function process(metadata: FileMetadata): Promise<void> {
   animateAndAnnotateLines(noVirtLineSelector, annotateLine);
 }
 
-async function promptPastReport(metadata: FileMetadata): Promise<void> {
```

Comment:
There will never be a need for this code anymore. This was called when the metadata for a branch gave the HEAD commit and there was no coverage for that HEAD commit. If you're on a branch now, Codecov will handle this on the API side and give you the last good coverage.